### PR TITLE
Added multinomial logistic regression

### PR DESCRIPTION
**Pull request by bouzaghrane**

### CODE REVIEW

**Comment by bouzaghrane** [comment]
@timothyb0912 @hassanobeid1994 whenever yall have a change, can you take a quick look at this?

File: notebooks/working/5.0-mab-simulation-causal-graph.md [hunk 1]
```diff
@@ -801,6 +802,24 @@ def FitAlternativeRegression(regressions, reg_types, data):
             
             # Store model
             regression_results[regressions[reg][1]+'_on_'+regressions[reg][0]] = results
+
+        # Multinomial logistic regression
+        if reg_types[reg] == 'multinomial_logistic':
+            # Prepare data and initialize model
+            # regressions[reg][0] COULD be a list
+            data_x = data[regressions[reg][0]]
```
Comment (timothyb0912) [inline notebooks/working/5.0-mab-simulation-causal-graph.md]: If `regressions[reg][0]` is a string, will line 815 work? I was of the impression that sklearn estimators need a 2d array for `X`. If so you should add a line that converts `data_x` to be 2D if necessary.

**Review by timothyb0912 — APPROVED** [review]
Besides my 1 comment below this looks good to me.

File: notebooks/working/5.0-mab-simulation-causal-graph.md (hunk 1 above)
Comment (bouzaghrane) [reply notebooks/working/5.0-mab-simulation-causal-graph.md]: Whoops! totally missed that. Thanks for catching that! I just made the change, check it out.

**Comment by bouzaghrane** [comment]
@timothyb0912 I made the changes bro. Check it out.

**Review by timothyb0912 — APPROVED** [review]

**Comment by timothyb0912** [comment]
@bouzaghrane feel free to merge this if there are no other issues to be addressed here.

**Comment by timothyb0912** [comment]
@bouzaghrane: Please merge or close...

**Pull request closed without merging**